internal/persistence/sql/migrations/sql/20230228091200000000_add-on-delete-cascade-to-relationship.up.sql

```diff
@@ -0,0 +1,4 @@
+ALTER TABLE
+  keto_relation_tuples DROP CONSTRAINT keto_relation_tuples_uuid_nid_fk,
+ADD
+  CONSTRAINT keto_relation_tuples_nid_fk FOREIGN KEY (nid) REFERENCES networks (id) ON UPDATE RESTRICT ON DELETE CASCADE;
```

Comment:
CockroachDB does not seem to like dropping constraints and re-creating them with the same name in the same transaction. So I renamed the constraint `keto_relation_tuples_nid_fk`, as it matches the table name better anyway.